### PR TITLE
Change of heading

### PR DESCRIPTION
**Pull request by schangccg**
for add logo and title ticket:
https://github.com/muccg/mtm/issues/234
https://github.com/muccg/mtm/issues/235

for hide sections in mtm ticket: 
https://github.com/muccg/mtm/issues/233

### CODE REVIEW

File: rdrf/rdrf/templatetags/project_title.py [hunk 1]
```diff
@@ -5,4 +5,7 @@
 
 @register.simple_tag
 def project_title():
-    return "%s" % (settings.PROJECT_TITLE)
+    try:
```
Comment (id2359) [inline rdrf/rdrf/templatetags/project_title.py]: You could test for attribute like this:   hasattr(settings, "PROJECT_TITLE")

File: rdrf/rdrf/templatetags/project_title.py [hunk 2]
```diff
@@ -5,4 +5,7 @@
...
+        return "%s" % (settings.PROJECT_TITLE)
+    except:
+        return "no_project_logo"
```
Comment (id2359) [inline rdrf/rdrf/templatetags/project_title.py]: if hasattr(settings,"PROJECT_TITLE"):
     return settings.PROJECT_TITLE

File: rdrf/rdrf/templatetags/project_logo.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+@register.simple_tag
+def project_logo():
```
Comment (id2359) [inline rdrf/rdrf/templatetags/project_logo.py]: put check here as below

File: rdrf/rdrf/patient_view.py [hunk 4]
```diff
@@ -505,23 +505,27 @@ def get(self, request, registry_code, patient_id):
...
             return redirect("%s?next=%s" % (login_url, patient_edit_url))
 
-        # Section variable hardcoded for now
-        section_blacklist=('Registry', 'Next of Kin')
+
+        registry_model = Registry.objects.get(code=registry_code)
+
+        if "section_blacklist" in registry_model.metadata:
+            section_blacklist = registry_model.metadata["section_blacklist"]
+            section_blacklist = [_("{}".format(x)) for x in section_blacklist]
```
Comment (id2359) [inline rdrf/rdrf/patient_view.py]: section_blacklist = [_(x) for x in section_blacklist]
or 

section_blacklist = map(_,section_blacklist)

File: rdrf/rdrf/patient_view.py [hunk 5]
```diff
@@ -712,14 +717,20 @@ def post(self, request, registry_code, patient_id):
...
-        if request.user.is_parent:
-            context['parent'] = ParentGuardian.objects.get(user=request.user)
+
+        if "section_blacklist" in registry_model.metadata:
+            section_blacklist = registry_model.metadata["section_blacklist"]
+            section_blacklist = [_("{}".format(x)) for x in section_blacklist]
+        else:
+            section_blacklist = []
+
+        context["section_blacklist"] = section_blacklist
```
Comment (id2359) [inline rdrf/rdrf/patient_view.py]: avoid duplicating code - 
extract a function of the registry model

File: rdrf/rdrf/patient_view.py (hunk 4 above)
Comment (schangccg) [reply rdrf/rdrf/patient_view.py]: Fixed.

File: rdrf/rdrf/patient_view.py (hunk 5 above)
Comment (schangccg) [reply rdrf/rdrf/patient_view.py]: Fixed.

File: rdrf/rdrf/templatetags/project_logo.py (hunk 3 above)
Comment (schangccg) [reply rdrf/rdrf/templatetags/project_logo.py]: Fixed.

File: rdrf/rdrf/templatetags/project_title.py (hunk 1 above)
Comment (schangccg) [reply rdrf/rdrf/templatetags/project_title.py]: Fixed.

**Pull request merged**